Add keyboard shortcuts for copying View, Scene, Field, and Object IDs in dev popover

ivojolo opened the pull request:
Added keyboard shortcuts to the KTL Dev Popover for faster ID copying:

  - **V** - Copy View ID
  - **S** - Copy Scene ID
  - **F** - Copy Field ID
  - **O** - Copy Object ID

These shortcuts only activate when the dev popover (activated via ctrl+shift) is visible and you're not typing in input fields. This allows a faster method of copying the ID instead of clicking on the copy buttons in the dev popover.

CSWinnall commented on KTL.js:
Hi @ivojolo Looks like the function has been duplicated in your code.

Craig

ivojolo replied:
Should be fixed now

cortexrd reviewed (changes requested):
Just a minor detail, Ivan:  I have set a standard of using single quotes throughout the KTL, by default.  Then, the second choice is the backticks when we need to use template literal (the ${variable} notation).

Can you replace them all please?

BTW, you can use your profile to tell Claude about this, if you want to apply this policy across the board and have all your code to use the same style.

ivojolo committed:
feat: Add keyboard shortcuts for copying View, Scene, Field, and Object IDs in dev popover

Added keyboard shortcuts to the KTL Dev Popover for faster ID copying:
- Press V to copy View ID
- Press S to copy Scene ID
- Press F to copy Field ID
- Press O to copy Object ID

These shortcuts only activate when the dev popover is visible and you're not typing in input fields. This improves developer efficiency when working with Knack's Builder.

ivojolo commented:
> Just a minor detail, Ivan: I have set a standard of using single quotes throughout the KTL, by default. Then, the second choice is the backticks when we need to use template literal (the ${variable} notation).
> 
> Can you replace them all please?
> 
> BTW, you can use your profile to tell Claude about this, if you want to apply this policy across the board and have all your code to use the same style.

The code should now follow your standards.
